content/en/docs/Features/istio-component-status.md

```diff
@@ -25,7 +25,7 @@ The certificates shown depends on how Istio is configured. The following cases a
 
 * Using Istio CA certificates (default), the information shown is from a secret named *istio-ca-secret*.
 * Using [Plug in CA certificates](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/), the information shown is from a secret named *cacerts*.
-* Using [DNS certificates](https://istio.io/latest/docs/tasks/security/cert-management/dns-cert/), the information shown is from reading many secrets found in Istio configuration.
+* Using [Custom CA with Kubernetes CSR](https://istio.io/latest/docs/tasks/security/cert-management/custom-ca-k8s/), the information shown is from a custom CA integrated with the Kubernetes CSR API.
```

Comment:
It seems that Istio removed support for DNS certificates in 1.14, they still are in 1.13: https://istio.io/v1.13/docs/tasks/security/cert-management/dns-cert/

Custom CA with Kubernetes CSR is not the same feature and we didn't implemented in Kiali yet, I guess we need to put the Istio's oldie link here and maybe mention someway it's not supported anymore in 1.14.

Reply:
Thanks @leandroberetta 
Adding change